Library/Formula/postgresql.rb

```diff
@@ -77,8 +82,22 @@ def install
   end
 
   def post_install
-    unless File.exist? "#{var}/postgres"
-      system "#{bin}/initdb", "#{var}/postgres"
+    pg_version = (var/"postgres/PG_VERSION").read.chomp
+
+    if (var/"postgres").exist? && (var/"postgres-#{pg_version}").exist?
+      puts "We have not attempted to migrate your data due to your multi-postgresql setup."
+    end
```

Comment:
This is just printing a message and continuing through the function. I'm also not sure what's meant here; why is this a "multi" setup and why can't we try to migrate this data?